cdap-docs/build.sh

```diff
@@ -182,24 +188,43 @@ function build_javadocs() {
     local debug_flag=''
   fi
   if [[ ${javadoc_type} == ${ALL} ]]; then
-    javadoc_run="javadoc:aggregate"
+    javadoc_run="javadoc:aggregate -P release"
   else
-    javadoc_run="site"
+    javadoc_run="site -P templates"
   fi
   local start=`date`
   cd ${PROJECT_PATH}
   MAVEN_OPTS="-Xmx4g -XX:MaxPermSize=256m" # match other CDAP builds
-  mvn clean package ${javadoc_run} -P examples,templates,release -DskipTests -Dgpg.skip=true -DisOffline=false ${debug_flag}
-  warnings=$?
-  if [[ ${warnings} -eq 0 ]]; then
-    echo
-    echo "Javadocs Build Start: ${start}"
-    echo "                 End: `date`"
+  local temp_repo="-Dmaven.repo.local=/tmp/repo-cdap-docs/${GIT_BRANCH}"
+  echo "Using temp_repo '${temp_repo}'"
+  if [[ -d ${temp_repo} ]]; then
+    rm -rf ${temp_repo}
+  fi 
```

Comment:
If you stick this under target somewhere, it shouldn't ever exist on a clean checkout, anyway.